pom.xml

```diff
@@ -1230,7 +1278,57 @@
     <profile>
       <id>release</id>
       <build>
-        <plugins>
+          <pluginManagement>
+              <plugins>
+                  <plugin>
+                      <artifactId>maven-scm-plugin</artifactId>
+                      <version>1.8.1</version>
+                      <configuration>
+                          <includes>pnc-ui/bower.json,pnc-ui/package.json,pnc-ui/npm-shrinkwrap.json</includes>
+                          <message>Update JSON Versions</message>
+                      </configuration>
+                  </plugin>
+              </plugins>
+          </pluginManagement>
+          <plugins>
+              <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>generate-compiler-sources</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <loadproperties srcFile="${executionRoot}/release.properties">
+                                    <filterchain>
+                                        <linecontains>
+                                            <contains value="project.rel.org.jboss.pnc\:parent"/>
+                                        </linecontains>
+                                    </filterchain>
+                                </loadproperties>
+
+                                <property name="newversion" value="${project.rel.org.jboss.pnc:parent}"/>
+
+                                <replaceregexp file="${basedir}/bower.json"
+                                               match="version&quot;: &quot;[0-9].*&quot;"
+                                               replace="version&quot;: &quot;${newversion}&quot;"/>
+                                <replaceregexp file="${basedir}/package.json"
+                                               match="version&quot;: &quot;[0-9].*&quot;"
+                                               replace="version&quot;: &quot;${newversion}&quot;"/>
+                                <replaceregexp file="${basedir}/npm-shrinkwrap.json"
```

Comment:
@rnc These replacements (L1316-1324) scare me a bit, there are tons of versions inside those files, and versions of resolved dependencies could potentially match the PNC version. But seems that only npm-shrinkwrap.json file has many  "version": "X.Y.Z" declarations, so package.json and bower.json should be fine. We could remove the npm-shrinkwrap.json file from the replacement or enforce a more strict replacement to be sure that only the pnc-ui version gets replaced.